distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -148,7 +146,11 @@ The default trust store and key store files for ${branding} included in `etc/key
 Self-signed certificates should never be used outside of development/testing areas.
 ====
 
-===== Certificate Revocation List (CRL)
+===== Managing Certificate Revocation List (CRL)
+
+* *{hardening-step}*
+
+For hardening purposes, it is recommended to implement a way to verify the CRL at least daily.
```

Comment:
Not sure how to comment on a line below that wasn't modified, but there is a duplicate file in the bullets line 184-188.